README.md

```diff
@@ -131,7 +143,17 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is an "Entity Relationship Diagram." It is basically a visual way to describe data and its relationship to each other. Key to ERDs is dependency, and visually showing what attributes are related to one another according to dependency.
+
+With regards to the example above:
+Person :name, :gender, :nationality
+The person may own a lamp and pet, and so they are all related
+  Lamp :material, :color
```

Comment:
Yes, this is true but specifically when a Pet is dependent on its Person owner, what the relationship? For example:

`belongs_to :person `